tools/grpc/genproto.sh

```diff
@@ -26,9 +27,7 @@ for pkg in ${INSTALL_PKGS}; do
 done
 
 GOBGP="${PWD}"
-GOPROTO="$(GO111MODULE=on go list -f '{{ .Dir }}' -m github.com/golang/protobuf)"
 
 echo "generating code"
-protoc -I "${GOPROTO}"/ptypes \
```

Comment:
There is no need to add this import path since these types ship with the protoc compiler under their proper include paths (google/protobuf). So, removing it.